packages/react-interactions/events/src/dom/testing-library/domEventSequences.js

```diff
@@ -18,6 +18,12 @@ function getPointerType(payload) {
   let pointerType = 'mouse';
   if (payload != null && payload.pointerType != null) {
     pointerType = payload.pointerType;
+  } else if (
+    Array.isArray(payload) &&
+    payload[0] != null &&
+    payload[0].pointerType != null
+  ) {
+    pointerType = payload[0].pointerType;
```

Comment:
If it's an array it must be a TouchEvent so we don't need to pluck it from the payload (or include a `pointerType` from the unit tests call sites)